### DISCUSSION TITLE
run-docker.sh errors

### DISCUSSION THREAD

**Discussion by zil70** [Xilinx/finn#459] [Q&A]
Hi,
  I have cloned 0.7 from depo. However, when I tried to run quicktest and I got the follow error messages:
./run-docker.sh quicktest
Please set PLATFORM_REPO_PATHS pointing to Vitis platform files (DSAs).
This is required to be able to use Alveo PCIe cards.
Running test suite (non-Vivado, non-slow tests)
Docker container is named finn_dev_bing
Docker tag is named xilinx/finn:v0.7-dirty.xrt_202010.2.7.766_18.04-amd64-xrt
Mounting /mnt/finn/finn/builddir into /mnt/finn/finn/builddir
Mounting /home/bing/Downloads/fpga/tools/Xilinx into /home/bing/Downloads/fpga/tools/Xilinx
Port-forwarding for Jupyter 8888:8888
Port-forwarding for Netron 8081:8081
Vivado IP cache dir is at /mnt/finn/finn/builddir/vivado_ip_cache
Using default PYNQ board Pynq-Z2
ERROR: Unable to find FINN source code in /workspace/finn
ERROR: Ensure you have passed -v <path-to-finn-repo>:/workspace/finn to the docker run command

Any idea?
Much thanks

**Comment by superpi15** [comment]
Could you run only: "./run-docker.sh"
This will enter a docker shell. 
Then you can use "ls" command to see the contents. 
I'm curious what's there.

**Comment by maltanar** [comment]
Can you add `echo "$DOCKER_EXEC"` into the `run-docker.sh` script just before the last line (so after the `DOCKER_EXEC+="$FINN_DOCKER_TAG $DOCKER_CMD"` line) and share what it prints out to the terminal? Looks like something is going wrong with the launching of the container.

**Comment by lloo099** [comment]
Pls set PLATFORM_REPO_PATHS first

**Reply by fpjentzsch** [reply]
This is only needed to build for Alveo, not for running the quicktest.

**Comment by Ba1tu3han** [comment]
Hello @zil70 , 

I met the same problem. Did you solve it? If yes, How did you do that? Could you share it?

Thank you,
